refarch-gateway/src/main/resources/application.yml

```
@@ -57,8 +57,12 @@ management:
-    name: ${spring.application.name}
-    version: '@project.version@'
+    name: @project.artifactId@
+    version: @project.version@
+    description: @project.description@
+  build:
+    java.version: @java.version@
+    spring-cloud.version: @spring-cloud-dependencies.version@
 
 config:
-  map5xxto400: true
+  map5xxto400: true
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)